+ source /root/mugen/libs/locallibs/common_lib.sh
++ python3 --version
++ '[' 0 -eq 0 ']'
++ source /root/mugen/libs/locallibs/common_lib_python.sh
+ main
++ type -t post_test
+ '[' -n function ']'
+ trap post_test EXIT INT HUP TERM
+ rpm -qa
+ grep expect
++ type -t config_params
+ '[' -n function ']'
+ config_params
+ LOG_INFO 'Start loading data!'
+ message='Start loading data!'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start loading data!'
Fri Apr 28 11:07:10 2023 - INFO  - Start loading data!
++ SSH_CMD 'ip route | grep 10.198.114.3 | awk '\''{print$3}'\''' 10.198.114.3 'openEuler12#$' root
++ tr '\r' ' '
++ cmd='ip route | grep 10.198.114.3 | awk '\''{print$3}'\'''
++ remoteip=10.198.114.3
++ remotepasswd='openEuler12#$'
++ remoteuser=root
++ timeout=300
++ connport=22
++ bash /root/mugen/libs/locallibs/sshcmd.sh -c 'ip route | grep 10.198.114.3 | awk '\''{print$3}'\''' -i 10.198.114.3 -u root -p 'openEuler12#$' -t 300 -o 22
++ tail -n 1
Fri Apr 28 11:07:11 2023 - WARN  - the remote user uses the default configuration.
Fri Apr 28 11:07:12 2023 - WARN  - the remote password uses the default configuration.
Fri Apr 28 11:07:12 2023 - WARN  - the connect port using the default configuration
++ ret=0
++ test 0 -ne 0
++ return 0
+ remote_eth1='eth0 '
++ echo eth0
+ remote_eth1=eth0
+ LOG_INFO 'Loading data is complete!'
+ message='Loading data is complete!'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Loading data is complete!'
Fri Apr 28 11:08:27 2023 - INFO  - Loading data is complete!
++ type -t pre_test
+ '[' -n function ']'
+ pre_test
+ LOG_INFO 'Start environment preparation.'
+ message='Start environment preparation.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start environment preparation.'
Fri Apr 28 11:08:27 2023 - INFO  - Start environment preparation.
+ SSH_CMD 'yum install -y samba; sed -i '\''/testsamba/d'\'' /etc/security/opasswd;useradd testsamba;
	(echo openEuler12#$;echo openEuler12#$) | smbpasswd -a testsamba -s' 10.198.114.3 'openEuler12#$' root
+ cmd='yum install -y samba; sed -i '\''/testsamba/d'\'' /etc/security/opasswd;useradd testsamba;
	(echo openEuler12#$;echo openEuler12#$) | smbpasswd -a testsamba -s'
+ remoteip=10.198.114.3
+ remotepasswd='openEuler12#$'
+ remoteuser=root
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c 'yum install -y samba; sed -i '\''/testsamba/d'\'' /etc/security/opasswd;useradd testsamba;
	(echo openEuler12#$;echo openEuler12#$) | smbpasswd -a testsamba -s' -i 10.198.114.3 -u root -p 'openEuler12#$' -t 300 -o 22
1 packets transmitted, 0 received, 100% packet loss, time 0ms
Fri Apr 28 11:08:38 2023 - ERROR - connection to 10.198.114.3 failed.
+ ret=101
+ test 101 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:101'
+ message='Failed in remote CMD operation:101'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:101'
Fri Apr 28 11:08:39 2023 - ERROR - Failed in remote CMD operation:101
+ return 101
+ SSH_CMD 'cp -a /etc/samba/smb.conf /etc/samba/smb.conf.bak;echo  \" \" >> /etc/samba/smb.conf;
	echo  \"\[testsamba\]\" >> /etc/samba/smb.conf;echo  \"\tcomment = public stuff\" >> /etc/samba/smb.conf;
	echo  \"\tpath = /home/testsamba\" >> /etc/samba/smb.conf' 10.198.114.3 'openEuler12#$' root
+ cmd='cp -a /etc/samba/smb.conf /etc/samba/smb.conf.bak;echo  \" \" >> /etc/samba/smb.conf;
	echo  \"\[testsamba\]\" >> /etc/samba/smb.conf;echo  \"\tcomment = public stuff\" >> /etc/samba/smb.conf;
	echo  \"\tpath = /home/testsamba\" >> /etc/samba/smb.conf'
+ remoteip=10.198.114.3
+ remotepasswd='openEuler12#$'
+ remoteuser=root
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c 'cp -a /etc/samba/smb.conf /etc/samba/smb.conf.bak;echo  \" \" >> /etc/samba/smb.conf;
	echo  \"\[testsamba\]\" >> /etc/samba/smb.conf;echo  \"\tcomment = public stuff\" >> /etc/samba/smb.conf;
	echo  \"\tpath = /home/testsamba\" >> /etc/samba/smb.conf' -i 10.198.114.3 -u root -p 'openEuler12#$' -t 300 -o 22
1 packets transmitted, 0 received, 100% packet loss, time 0ms
Fri Apr 28 11:08:51 2023 - ERROR - connection to 10.198.114.3 failed.
+ ret=101
+ test 101 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:101'
+ message='Failed in remote CMD operation:101'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:101'
Fri Apr 28 11:08:51 2023 - ERROR - Failed in remote CMD operation:101
+ return 101
+ SSH_CMD 'systemctl start smb;systemctl enable smb;systemctl stop firewalld;
	setsebool samba_export_all_ro on;setsebool samba_export_all_rw on;chmod 755 /home/testsamba' 10.198.114.3 'openEuler12#$' root
+ cmd='systemctl start smb;systemctl enable smb;systemctl stop firewalld;
	setsebool samba_export_all_ro on;setsebool samba_export_all_rw on;chmod 755 /home/testsamba'
+ remoteip=10.198.114.3
+ remotepasswd='openEuler12#$'
+ remoteuser=root
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c 'systemctl start smb;systemctl enable smb;systemctl stop firewalld;
	setsebool samba_export_all_ro on;setsebool samba_export_all_rw on;chmod 755 /home/testsamba' -i 10.198.114.3 -u root -p 'openEuler12#$' -t 300 -o 22
1 packets transmitted, 0 received, 100% packet loss, time 0ms
Fri Apr 28 11:09:02 2023 - ERROR - connection to 10.198.114.3 failed.
+ ret=101
+ test 101 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:101'
+ message='Failed in remote CMD operation:101'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:101'
Fri Apr 28 11:09:03 2023 - ERROR - Failed in remote CMD operation:101
+ return 101
+ DNF_INSTALL cifs-utils
+ pkgs=cifs-utils
+ node=1
+ '[' -z '' ']'
+ tmpfile=
++ python3 /root/mugen/libs/locallibs/rpm_manage.py install --pkgs cifs-utils --node 1 --tempfile ''
+ tmpfile2=/tmp/tmprj00e7qp
+ '[' -z '' ']'
+ tmpfile=/tmp/tmprj00e7qp
+ systemctl stop firewalld
Failed to stop firewalld.service: Unit firewalld.service not loaded.
+ mkdir -p /home/client
+ LOG_INFO 'Environmental preparation is over.'
+ message='Environmental preparation is over.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Environmental preparation is over.'
Fri Apr 28 11:09:37 2023 - INFO  - Environmental preparation is over.
++ type -t run_test
+ '[' -n function ']'
+ run_test
+ LOG_INFO 'Start executing testcase.'
+ message='Start executing testcase.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start executing testcase.'
Fri Apr 28 11:09:38 2023 - INFO  - Start executing testcase.
+ mount -t cifs -o 'username=testsamba,password=openEuler12#$' //10.198.114.3/testsamba /home/client
mount error: cifs filesystem not supported by the system
mount error(19): No such device
Refer to the mount.cifs(8) manual page (e.g. man mount.cifs) and kernel log messages (dmesg)
+ CHECK_RESULT 32
+ actual_result=32
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 32 ']'
+ '[' 0 -eq 0 ']'
+ test 32x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_storage_smb_network_latency.sh line 48'
+ message='oe_test_storage_smb_network_latency.sh line 48'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_storage_smb_network_latency.sh line 48'
Fri Apr 28 11:09:39 2023 - ERROR - oe_test_storage_smb_network_latency.sh line 48
+ '[' 0 -eq 1 ']'
+ return 0
+ df -h
+ grep 10.198.114.3
+ grep -i /home/client
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_storage_smb_network_latency.sh line 50'
+ message='oe_test_storage_smb_network_latency.sh line 50'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_storage_smb_network_latency.sh line 50'
Fri Apr 28 11:09:40 2023 - ERROR - oe_test_storage_smb_network_latency.sh line 50
+ '[' 0 -eq 1 ']'
+ return 0
+ SSH_CMD 'tc qdisc add dev eth0 root netem delay 300ms' 10.198.114.3 'openEuler12#$' root
+ cmd='tc qdisc add dev eth0 root netem delay 300ms'
+ remoteip=10.198.114.3
+ remotepasswd='openEuler12#$'
+ remoteuser=root
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c 'tc qdisc add dev eth0 root netem delay 300ms' -i 10.198.114.3 -u root -p 'openEuler12#$' -t 300 -o 22
Fri Apr 28 11:09:41 2023 - WARN  - the remote user uses the default configuration.
Fri Apr 28 11:09:42 2023 - WARN  - the remote password uses the default configuration.
Fri Apr 28 11:09:43 2023 - WARN  - the connect port using the default configuration
spawn ssh -o ConnectTimeout=300 -p 22 root@10.198.114.3 tc qdisc add dev eth0 root netem delay 300ms
root@10.198.114.3's password: 
Error: Specified qdisc kind is unknown.
+ ret=2
+ test 2 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:2'
+ message='Failed in remote CMD operation:2'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:2'
Fri Apr 28 11:09:55 2023 - ERROR - Failed in remote CMD operation:2
+ return 2
+ CHECK_RESULT 2
+ actual_result=2
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 2 ']'
+ '[' 0 -eq 0 ']'
+ test 2x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_storage_smb_network_latency.sh line 52'
+ message='oe_test_storage_smb_network_latency.sh line 52'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_storage_smb_network_latency.sh line 52'
Fri Apr 28 11:09:56 2023 - ERROR - oe_test_storage_smb_network_latency.sh line 52
+ '[' 0 -eq 1 ']'
+ return 0
+ SSH_CMD 'echo hello > /home/testsamba/test;chmod 755 /home/testsamba/test' 10.198.114.3 'openEuler12#$' root
+ cmd='echo hello > /home/testsamba/test;chmod 755 /home/testsamba/test'
+ remoteip=10.198.114.3
+ remotepasswd='openEuler12#$'
+ remoteuser=root
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c 'echo hello > /home/testsamba/test;chmod 755 /home/testsamba/test' -i 10.198.114.3 -u root -p 'openEuler12#$' -t 300 -o 22
1 packets transmitted, 0 received, 100% packet loss, time 0ms
Fri Apr 28 11:10:07 2023 - ERROR - connection to 10.198.114.3 failed.
+ ret=101
+ test 101 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:101'
+ message='Failed in remote CMD operation:101'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:101'
Fri Apr 28 11:10:08 2023 - ERROR - Failed in remote CMD operation:101
+ return 101
+ grep hello /home/client/test
grep: /home/client/test: No such file or directory
+ CHECK_RESULT 2
+ actual_result=2
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 2 ']'
+ '[' 0 -eq 0 ']'
+ test 2x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_storage_smb_network_latency.sh line 55'
+ message='oe_test_storage_smb_network_latency.sh line 55'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_storage_smb_network_latency.sh line 55'
Fri Apr 28 11:10:09 2023 - ERROR - oe_test_storage_smb_network_latency.sh line 55
+ '[' 0 -eq 1 ']'
+ return 0
+ LOG_INFO 'End of testcase execution.'
+ message='End of testcase execution.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'End of testcase execution.'
Fri Apr 28 11:10:09 2023 - INFO  - End of testcase execution.
+ CASE_RESULT 0
+ case_re=0
+ test -z 4
+ test 4 -gt 0
+ LOG_ERROR 'failed to execute the case.'
+ message='failed to execute the case.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'failed to execute the case.'
Fri Apr 28 11:10:10 2023 - ERROR - failed to execute the case.
+ exit 4
+ post_test
+ LOG_INFO 'start environment cleanup.'
+ message='start environment cleanup.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'start environment cleanup.'
Fri Apr 28 11:10:11 2023 - INFO  - start environment cleanup.
+ umount /home/client
umount: /home/client: not mounted.
+ rmdir /home/client
rmdir: failed to remove '/home/client': Directory not empty
+ SSH_CMD 'tc qdisc del dev eth0 root netem delay 300ms; systemctl stop smb;
    rm -f /etc/samba/smb.conf;mv /etc/samba/smb.conf.bak /etc/samba/smb.conf;yum remove samba -y;
    userdel -r testsamba; systemctl start firewalld' 10.198.114.3 'openEuler12#$' root
+ cmd='tc qdisc del dev eth0 root netem delay 300ms; systemctl stop smb;
    rm -f /etc/samba/smb.conf;mv /etc/samba/smb.conf.bak /etc/samba/smb.conf;yum remove samba -y;
    userdel -r testsamba; systemctl start firewalld'
+ remoteip=10.198.114.3
+ remotepasswd='openEuler12#$'
+ remoteuser=root
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c 'tc qdisc del dev eth0 root netem delay 300ms; systemctl stop smb;
    rm -f /etc/samba/smb.conf;mv /etc/samba/smb.conf.bak /etc/samba/smb.conf;yum remove samba -y;
    userdel -r testsamba; systemctl start firewalld' -i 10.198.114.3 -u root -p 'openEuler12#$' -t 300 -o 22
1 packets transmitted, 0 received, 100% packet loss, time 0ms
Fri Apr 28 11:10:22 2023 - ERROR - connection to 10.198.114.3 failed.
+ ret=101
+ test 101 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:101'
+ message='Failed in remote CMD operation:101'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:101'
Fri Apr 28 11:10:23 2023 - ERROR - Failed in remote CMD operation:101
+ return 101
+ DNF_REMOVE
+ node=1
+ pkg_list=
+ mode=0
+ [[ -z /tmp/tmprj00e7qp ]]
+ '[' 0 -ne 0 ']'
+ '[' 1 == 0 ']'
+ python3 /root/mugen/libs/locallibs/rpm_manage.py remove --node 1 --pkgs '' --tempfile /tmp/tmprj00e7qp
+ '[' 0 -ne 0 ']'
+ systemctl start firewalld
Failed to start firewalld.service: Unit firewalld.service not found.
+ LOG_INFO 'Finish environment cleanup.'
+ message='Finish environment cleanup.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Finish environment cleanup.'
Fri Apr 28 11:10:35 2023 - INFO  - Finish environment cleanup.
